<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RiskCtrlWin</class>
 <widget class="QMainWindow" name="RiskCtrlWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>风控模块</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>净资产 &lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RiskCtrlBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>监控</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NetAssetLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RealNetAssetLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>实时净资产：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearPosiBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清仓</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="RiskCtrlStatusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
